languages/zh-CN/persona.json

```diff
@@ -0,0 +1,4 @@
+{
+    "mobile-menu-side": "Switch which side each mobile menu is on",
```

Comment:
Any reason mobile-menu-side isn't translated?

Reply:
I will resubmit a PR to translate this. tks!